openedx/core/djangoapps/oauth_dispatch/migrations/0003_application_data.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-06-05 13:19
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations
+
+
+def migrate_application_data(apps, schema_editor):
+    """
+    Migrate existing DOT Application models to new ScopedApplication models.
+    """
+    Application = apps.get_model(settings.OAUTH2_PROVIDER_APPLICATION_MODEL)
+    ScopedApplication = apps.get_model('oauth_dispatch', 'ScopedApplication')
+
+    for application in Application.objects.all():
+        ScopedApplication.objects.get_or_create(
```

Comment:
Would it make sense to use [update_or_create](https://docs.djangoproject.com/en/2.0/ref/models/querysets/#update-or-create) instead?